android/expoview/src/main/java/host/exp/exponent/experience/ExperienceActivity.java

```diff
@@ -357,6 +354,10 @@ protected void onDoneLoading() {
     Analytics.sendTimedEvents(mManifestUrl);
   }
 
+  public void onEvent(BaseExperienceActivity.ExperienceContentLoaded event) {
+    mLoadingProgressPopupController.hide();
+  }
```

Comment:
This one is going to be refactored more in next PRs addressing `expo-splash-screen` integration.